backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+include::_attributes/common-attributes.adoc[]
+= Introduction to {oadp-full}
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+
+The {oadp-first} product safeguards customer applications on {product-title}. OADP provides disaster recovery protection for {product-title} applications, application-related cluster resources, persistent volumes, and internal images.
+
+OADP can be used to back up containerized applications and virtual machines (VMs).
+
+OADP cannot be used as a disaster recovery solution for etcd or OpenShift Operators. For more information on etcd, see xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[Backing up etcd].
+
+OADP provides APIs to enable multiple approaches to customize a backup to avoid backing up inappropriate resources.
+
+OADP provides the following APIs:
+
+* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[Backup]
+* xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#restoring-applications[Restore]
+* xref:../../backup_and_restore/application_backup_and_restore/oadp-api.adoc#oadp-api[Schedule]
+* xref:../../backup_and_restore/application_backup_and_restore/oadp-api.adoc#oadp-api[BackupStorageLocation]
+* VolumeSnapshotLocation
```

Comment:
@anarnold97 I am assuming here that the link for  `BackupSnapshotLocation` is the same as for `VolumeSnapshotLocation`. 
I am not sure which option would be less confusing, leaving `VolumeSnapshotLocation` without a link, or provide the same link for both. I defer to your guidance.

Reply:
Thanks @mburke5678 - will make this change later this evening